.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 build
-coverage
\ No newline at end of file
+coverage
+.idea
```

Comment:
There shouldn't be local development folders in the .gitignore. My preference would be to put this in your global .gitignore. https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/